app/views/public/faq.en.html.erb

```diff
@@ -340,6 +341,20 @@
 
 <hr />
```

Comment:
This `<hr />` should go below the whole section, so `Gross ROI` is in the wrong place